### DISCUSSION TITLE
커스텀 애니메이션 적용이 잘 안되네요

### DISCUSSION THREAD

**Discussion by HanjuKim786** [naverz/zepeto-studio-kor#817] [5. Q&A]
3d max로 커스텀 애니메이션(포즈)를 잡고 유니티에서 휴머노이드 설정한 후에 적용해 봤더니 3d maxs 상에 잡았던 포즈와 다르게 표현됩니다.

원인을 알 수 없어 문의를 남깁니다.

의자에 앉아있는 포즈인데, 팔다리가 마치 블랜딩 된것처럼 회전이 덜 된 포즈로 계속 나오네요.
애니메이터에서 블랜딩 수치를 0으로 줘도 똑같이 나옵니다.

혹시 휴머노이드에서 아바타를 설정해 줘야 할까요? 아바타를 설정해주고 싶어도 에러가 떠서 아바타 설정을 못하고 휴머노이드만 설정해 놓고 하는 중입니다.

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 도움을 드리고 싶은데 글만 봐서는 파악이 어렵습니다.
관련 fbx 파일을 첨부해주시면 확인해 보겠습니다. 
감사합니다.

**Reply by HanjuKim786** [reply]
[anim_sit.Zip](https://github.com/naverz/zepeto-studio-kor/files/8795409/anim_sit.Zip)

파일 첨부했습니다. 답변감사합니다.

**Reply by Zepeto-TechEvangelist** [reply]
3D max로 커스텀 애니메이션 제작하실때 어떤 리그를 사용하셨는지 알려주실 수 있나요?

**Reply by HanjuKim786** [reply]
https://studio.zepeto.me/guides/creating-your-item-3d-modelers-1
위 URL에서 creator base set 을 다운로드 받아 사용했습니다.

**Reply by HanjuKim786** [reply]
위 링크의 가이드 문서에 안내되어 있는 리깅된 파일로는 헤어본, 기타 본이 없는것 같습니다. 이 가이드 문서에 나와있는 리깅파일로 아바타를 만들어서 사용해도 문제되는 부분이 없을까요? 아니면 모든 본이 포함된 fbx 최신 버전을 제공받을 수 있을까요?

**Reply by Zepeto-TechEvangelist** [reply]
안녕하세요, 사용하신 파일은 마야 기반입니다.
아이템 가이드에 올라가 있는 파일을 사용해서 3d max 애니메이션 제작하시다보니 의도와 다르게 나온것 같습니다. 
커스텀 애니메이션을 위한 가이드라인을 준비하고 있으니 조금만 기다려 주시기를 부탁 드립니다.
그 전에는 mixamo등의 사이트에서 커스텀 애니메이션을 활용하는 방법이 있습니다. https://github.com/naverz/zepeto-studio-kor/discussions/538
감사합니다.
